go/runtime/host/sgx/ecdsa.go

```diff
@@ -193,7 +193,7 @@ func (ec *teeStateECDSA) Update(ctx context.Context, sp *sgxProvisioner, conn pr
 		return fresh, nil
 	}
 	var tcbBundle *pcs.TCBBundle
-	for _, update := range []pcs.UpdateType{pcs.UpdateStandard, pcs.UpdateEarly} {
+	for _, update := range []pcs.UpdateType{pcs.UpdateEarly, pcs.UpdateStandard} {
```

Comment:
Should `tcbCache` distinguish between standard and early TCB info?

Reply:
I don't think this is needed. Eventually a more advanced cache will be on-chain.